docs/docs/APIdocs/layers/nn/Pooling Layers/VolumetricMaxPooling.md

```diff
@@ -0,0 +1,122 @@
+## VolumetricMaxPooling ##
+
+**Scala:**
+```scala
+val layer = VolumetricMaxPooling[T](
+  kernelT, kernelW, kernelH,
+  strideT, strideW, strideH,
+  paddingT, paddingW, paddingH
+)
+```
+**Python:**
+```python
+layer = VolumetricMaxPooling(
+  kernelT, kernelW, kernelH,
+  strideT, strideW, strideH,
+  paddingT, paddingW, paddingH
+)
+```
+
+Applies 3D max-pooling operation in kT x kW x kH regions by step size dT x dW x dH steps.
```

Comment:
a little confuse about this expression.  do you mean "by step size dT x dW x dH" without the last "steps"?